addons/stock/models/product.py

```diff
@@ -597,37 +590,47 @@ def write(self, vals):
                 raise UserError(_("You can not change the type of a product that is currently reserved on a stock move. If you need to change the type, you should first unreserve the stock move."))
         return super(ProductTemplate, self).write(vals)
 
-    def action_update_quantity_on_hand(self):
-        default_product_id = self.env.context.get('default_product_id', self.product_variant_id.id)
-        if self.env.user.user_has_groups('stock.group_stock_multi_locations') or (self.env.user.user_has_groups('stock.group_production_lot') and self.tracking != 'none'):
-            product_ref_name = self.name + ' - ' + datetime.today().strftime('%m/%d/%y')
-            ctx = {'default_filter': 'product', 'default_product_id': default_product_id, 'default_name': product_ref_name}
+    def action_open_quants(self, products=None):
```

Comment:
It should be on product.product and self should be the product args

Reply:
It's better and more logical indeed, that's done, thanks !